pyanaconda/modules/common/errors/localization.py

```diff
@@ -0,0 +1,27 @@
+#
```

Comment:
Can't we just reuse, KeyboardConfigurationError from pyanaconda/modules/common/errors/configuration.py?
Seems generic enough.

Reply:
What would be the benefit? Seems to me that more verbose is better isn't it?

Reply:
I see your point. Errors are not split by the modules as structures etc... In that case we can avoid creating the `localization` error python module.